noxfile.py

```diff
@@ -108,9 +109,24 @@ def unit_noextras(session):
     default(session, install_extras=False)
 
 
+@nox.session(python=DEFAULT_PYTHON_VERSION)
+def mypy(session):
```

Comment:
Can you also add `mypy` to `nox.options.sessions` so that it runs in Kokoro?

Reply:
Ah, I thought it would be picked automatically/ Sure thing, will add. :+1: